racket/collects/racket/fasl.rkt

```diff
@@ -1,13 +1,17 @@
 #lang racket/base
 (require (for-syntax racket/base)
          "private/truncate-path.rkt"
-         "private/relative-path.rkt")
+         "private/relative-path.rkt"
+         (rename-in racket/base [write-byte r:write-byte]))
 
 (provide s-exp->fasl
          fasl->s-exp)
 
 ;; ----------------------------------------
 
+(define (write-byte byte out)
```

Comment:
Can you do this for `write-bytes` as well?

Reply:
Also, it would be good to have a comment explaining the reason for the wrapper, since it looks pointless at first glance.

FWIW, I could have changed all the `write-byte`s to some other name instead of renaming `write-byte` on import. But if that choice doesn't seem better to you, this way seems fine to me.

Reply:
I suggested this approach because it makes it harder to misuse, since you'd have to write `r:write-bytes` to do the wrong thing.

Reply:
That makes sense!